### PR TITLE
Final Student HTML Grading

### PR DESCRIPTION
**Pull request by vaishnavisowany**
This is the one. Please ignore the previous two pull requests!! Sorry about that!!

### CODE REVIEW

File: app/student/html/upload2.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+		*/
+		if($result) {
+      $abc = "Your file ".$fileName." has been successfully uploaded";
+     echo "<script type='text/javascript'>alert('$abc');window.open('index.php','_self',false);</script>";
+    //  echo "Your file <html><b><i>".$fileName."</i></b></html> has been successfully uploaded";
+			$query = "INSERT INTO html_assignment(assignment_file) VALUES ('$file')";
+			$link->query($query) or die("Error : ".mysqli_error($link));
+		}
+		else {
+      $emfe = "Sorry !!! There was an error in uploading your file";
```
Comment (wdickerson) [inline app/student/html/upload2.php]: I always end up here when uploading a file :(

**Comment by wdickerson** [comment]
Hey @VaishnaviSowany , I don't mind merging this, but take a look at the inline comment. 

Have you tried running this on multiple PCs? Maybe the destination folder doesn't exist on my PC.
Have you tried running this with the app connected to the AWS database? Manny gave instructions on this last class.

Let us know if you want us to merge as-is

**Comment by vaishnavisowany** [comment]
Okay. Let me check and get back to you! Thank you for the feedback!


Vaishnavi
HTML Grading Team

Sent from Mail for Windows 10

**Comment by vaishnavisowany** [comment]
Hey Will,

I checked my upload file and it is uploading in the temporary location. Which means that it shows in the folder. B ut, it is not uploading in the database. It used to upload in the local database. But, it is not uploading in the AWS database. I changed the config.php file too but it is not working. Do you have any suggestions?


Vaishnavi

Sent from Mail for Windows 10

**Comment by wdickerson** [comment]
Hmm, i would suggest that you start with a fresh folder on your PC and clone your repository into it, then start debugging from there with `config.php` pointing to the AWS database

I'm still wondering if this `Files/` folder where you are saving the files to does not exist on my PC. If that's the case, you might need to change where the files go. (but that is just a guess)

**Comment by vaishnavisowany** [comment]
Hey,

I will follow with the first suggestion.
Regarding the second one, I changed the name of the folder for other PCs and it still worked with the local database. But, since changing the database, it says that the file was uploaded but the file does not reflect in the database. I tried running the SQL individually and it does work. I will keep working on it. Thanks for the feedback.

Vaishnavi

Sent from Mail for Windows 10

**Comment by vaishnavisowany** [comment]
Hey Will,

I changed the code a bot so that the folder problem is solved as well as the upload problem. Please let me know if you find any other problems.

Vaishnavi

**Comment by wdickerson** [comment]
Hi Vaishavi, the upload file works, but something is strange... 
I logged in as a student and selected HTML Assignment 1. Then, I uploaded a file called "force.html".
Now, it looks like there is a _new assignment_ called "force.html"
See the screenshot

screen shot 2017-12-17 at 7 09 53 pm

I'll merge this now, and if you make any changes, you can open a new PR

**Pull request merged**